src/MagnumPlugins/StbResizeImageConverter/StbResizeImageConverter.cpp

```diff
@@ -34,9 +35,9 @@
 #include <Magnum/Math/ConfigurationValue.h>
 #include <Magnum/Trade/ImageData.h>
 
+#define STBIR_ASSERT CORRADE_INTERNAL_ASSERT
```

Comment:
To avoid accidental perf regressions in release builds, I'd make this a `CORRADE_INTERNAL_DEBUG_ASSERT`, for consistency with standard `assert()`.